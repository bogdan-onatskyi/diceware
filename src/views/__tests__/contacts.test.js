import React from 'react';

import Contacts from "../contacts";

describe("Contacts component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <Contacts/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should not have any props", () => {
        expect(Contacts.propTypes).toBe(undefined);
    });
});