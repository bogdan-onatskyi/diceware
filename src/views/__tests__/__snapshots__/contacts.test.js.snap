// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts component should match saved snapshot 1`] = `
<div
  className="app"
>
  <Header
    text="Контакты"
  />
  <Menu />
  <div
    className="app__content app__content--contacts"
  >
    <div
      className="app__content app__content--contacts-box"
    >
      <span
        className="row"
      >
        <span
          className="col1"
        >
          Телефон: 
        </span>
        <span
          className="col2"
        >
          +38 (097) 499‑73‑82
        </span>
      </span>
      <span
        className="row"
      >
        <span
          className="col1"
        >
          e‑mail: 
        </span>
        <a
          className="col2"
          href="mailto:gentoo.user@ukr.net"
        >
          gentoo.user@ukr.net
        </a>
      </span>
      <span
        className="row"
      >
        <span
          className="col1"
        >
          резюме: 
        </span>
        <a
          className="col2"
          href="https://www.work.ua/jobseeker/my/resumes/view/?id=4308769"
        >
          www.work.ua
        </a>
        <span
          className="col1"
        />
        <a
          className="col2"
          href="https://rabota.ua/jobsearch/cvbuilder?resumeId=11349311"
        >
          rabota.ua
        </a>
      </span>
    </div>
  </div>
  <Footer />
</div>
`;
